volume/nfs/nfs.go

```diff
@@ -98,11 +100,12 @@ func (d *NFSDriver) Get(volumeName string) (volume.Volume, error) {
 		driver: d,
 		tenant: getTenant(volumeName),
 	}
-	//actual NFS mount
-	if err := mount(volumeName, volumePath); err != nil {
-		return nil, err
+	if !d.networkDisabled {
```

Comment:
Mounting will create the directory if it isn't there and then mount the remote FS.  What  creates the directory on the local filesystem if the network is disabled?